db/seeds/tutorials.yml

```diff
@@ -297,6 +297,167 @@
       content: >
         After you hit submit, email the Transportation Office you selected for counseling and let them know that you've created a PPM shipment in DPS. They may need additional information from you before your shipment is approved.
 
+- title: Dual Military (Mil-to-Mil) Move
+  tutorial_steps:
+    - image_path: tutorials/dual-military-move-1.jpg
+      content: >
+        A Dual Move, also known as Mil-to-Mil, is when your spouse or significant other is also a service member or government civilian and you both have Orders to PCS.<br><br>
+        Check out the <a href="/faqs">FAQ</a> for details about weight allowances, and examples of Mil-to-Mil move scenarios. Usually, the higher ranking member fills out DPS.<br><br>
+        First, log into <a href="https://eta.sddc.army.mil/ETASSOPortal/default.aspx">ETA</a>, and choose DPS from your list of approved applications.
+      pro_tip: >
+        <strong>Pro-Tip:</strong> Turn off your pop-up blocker, otherwise the DPS window won't be able to open.
+    - image_path: tutorials/dual-military-move-2.jpg
+      content: >
+        If you haven’t logged in in a while you’ll be prompted to review your profile. Take this opportunity to confirm that everything is up to date.
+      pro_tip: >
+        <strong>Pro-Tips:</strong>
+        <ul>
+          <li>Save time by making sure you have a copy of your Orders. If you don't, ask your command for a Letter in Lieu, which can be used until your official orders are cut. Make sure you send your Transportation Office a copy of your Orders so that they can update the system.</li>
+          <li>You'll also be asked detailed information about special items you might ship, like the make and model of your Motorcycle, and even the calibre and serial number of each of your firearms. Get this information organized now to save time later.</li>
+          <li>You don't have to have an exact address at your new location to start the move process. Don't wait to schedule your move, peak season can mean your prefered move dates are taken, and lines are long at your local Transportation Office.</li>
```

Comment:
Run on sentence here (with some misspelling sprinkled in). What do you think about:

> Don't wait to schedule your move! The longer you wait, the more likely that your preferred move date will be unavailable, especially if it's during Peak Season. Lines can also get long at your local Transportation Office.